fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/provider/impl/S3Provider.java

```diff
@@ -171,10 +171,16 @@ private static AmazonS3 getClient(boolean iam, String cosApiKeyProperty, String
      * @return
      */
     public boolean exists() {
```

Comment:
I wonder if we should update the javadoc to explain that it doesn't actually return false when the bucket doesn't exist now...

Reply:
Updated